src/wasm/cpp/.gitignore

```diff
@@ -1,2 +1 @@
 output.wasm
-*.wat
```

Comment:
Do we want to check in wat files?

Reply:
I don't think so, and I'm not routinely generating them now so I don't need them ignored.